css/src/tokens/colors.scss

```diff
@@ -16,6 +16,9 @@ $box-shadow-color-extra-heavy: var(--theme-box-shadow-extra-heavy);
 
 $body-background: var(--theme-body-background);
 $body-background-medium: var(--theme-body-background-medium);
+$body-background-accent: var(--theme-body-background-accent);
+
+$card-background: var(--theme-card-background);
```

Comment:
rm card bg? we have no cards yet

Reply:
I thought I'd keep at least a variable for easier usage in docs right now.

Reply:
nvm, removed!